data/default_imsim_configs

```diff
@@ -50,4 +50,4 @@ sort_magnorm = True
 # FWHM in arcsec of the Gaussian to convolve with the baseline
 # AtmosphericPSF + OptWF PSF to account for additional instrumental
 # effects.
-gaussianFWHM = 0.3
+gaussianFWHM = 0.4
```

Comment:
Do we have the results in a histogram from the multiple visits you ran of ellipticity etc from Javier yet, to know if this is the value we want to use?

Reply:
We had an extensive discussion on this at XWG computing group session on Wednesday of the DESC meeting, and everyone agreed that this was the value we should use.